tests/ci_install.sh

```diff
@@ -33,27 +32,28 @@ if [[ "${RUNNER_OS}" == "Windows" ]] && [[ ${dependencies} != "minimal" ]]; then
     # keep in sync: setup.cfg
     while read requirement; do conda install --yes $requirement; done < tests/windows_conda_requirements.txt
 else
+    # upgrading pip to guarantee installing extra dependencies with [full] is supported
+    # this is only necessary for some versions of Python 3.8 and 3.9
+    # see https://github.com/yt-project/yt/issues/4270
     python -m pip install --upgrade pip
-    python -m pip install --upgrade wheel
-    python -m pip install --upgrade setuptools
 fi
 
 # Step 2: install deps and yt
 # installing in editable mode so this script may be used locally by developers
 # but the primary intention is to embed this script in CI jobs
 if [[ ${dependencies} == "minimal" ]]; then
     # test with minimal versions of runtime dependencies
-    python -m pip install -e .[test,minimal]
+    python -m pip install -e ".[test,minimal]"
 elif [[ ${dependencies} == "full" ]]; then
     # test with all optional runtime dependencies
 
     # this is required for cartopy. see
     # https://scitools.org.uk/cartopy/docs/latest/installing.html?highlight=install#building-from-source
```

Comment:
does this comment need updating?

Reply:
It doesn't. Actually it's the linked documentation that's out of date (see https://github.com/yt-project/yt/pull/4311#pullrequestreview-1271134788) ;-)